common/static/common/js/discussion/views/discussion_inline_view.js

```diff
@@ -118,7 +118,7 @@
             this.newPostView = new NewPostView({
                 el: this.newPostForm,
                 collection: this.discussion,
-                course_settings: this.course_settings,
+                course_settings: this.courseSettings,
```

Comment:
@noraiz-anwar You are right it could be good, but then we would have to change the view and subsequent tests. My initial priority was to fix the problem. 

But I got your point.